apps.json

```diff
@@ -668,14 +668,22 @@
         ],
         "url": "https://github.com/YunoHost-Apps/distbin_ynh"
     },
+    "django_example_ynh": {
+        "category": "dev",
+        "state": "working",
+        "subtags": [
+            "programming"
+        ],
+        "url": "https://github.com/YunoHost-Apps/django_example_ynh"
+    },
     "django-fmd": {
         "category": "iot",
         "level": 7,
         "state": "working",
         "url": "https://github.com/YunoHost-Apps/django-fmd_ynh"
     },
     "django-for-runners": {
-        "state": "inprogress",
+        "state": "working",
```

Comment:
```suggestion
        "category": "small_utilities",
        "state": "working",
```

Reply:
Let's add a category to Django-for-Runners, otherwise LGTM!

Reply:
i searched around for a suitable category, but there is no one. "small_utilities" seems to be wrong ;) Maybe "office" ?

About [django-for-runners](https://github.com/jedie/django-for-runners): 

> Store your GPX tracks of your running (or other sports activity) in django.

Reply:
I agree it is not perfect, but that's where we put Owntracks and µLogger.

Reply:
Anyways, that's unrelated to the aim of this PR. I dunno why your commit for Runners ended up here.

Reply:
OK. I create a new PR: https://github.com/YunoHost/apps/pull/1497